### ISSUE TITLE
Sateless components displayName is undefined

### ISSUE THREAD

**Issue by acondiff** [reactjs/react-docgen#247]
Here is the component in questions...

```
import React from 'react'
import PropTypes from 'prop-types'

const gutterWidth = 24

/**
 * Inside a Grid component, the Row ensures contained Cols will be on a different row
 */

export const Row = ({ children, style }) =>
  <div style={{
    marginTop: '0',
    marginRight: -gutterWidth / 2,
    marginBottom: '0',
    marginLeft: -gutterWidth / 2,
    display: 'flex',
    flex: '1',
    flexWrap: 'wrap',
    ...style
  }}>{children}</div>

Row.propTypes = {

  /**
   * Col's rendered within the Row
   */
  children: PropTypes.node,

  /**
   * Style override
   */
  style: PropTypes.object
}

export default Row
```

It outputs the following (notice that it is missing the `displayName`).
```
[ { description: 'Inside a Grid component, the Row ensures contained Cols will be on a different row',
    methods: [],
    props:
      { color: [Object],
        name: [Object],
        size: [Object],
        style: [Object] } } ]
```

**Comment by matheussampaio** [comment]
I also have this problem. It would be **really* nice if we use the variable name as `displayName`.

**Comment by danez** [comment]
This is fixed in c81a4bc98cb5dff48da0dc2e300270f2bd178586

Will make a new release soon.
